### PR TITLE
Adds a New Food: Ratatouille

### PR DESCRIPTION
**Pull request by t-toasted**
## About the PR 
As said in the title: adds in ratatouille to the game. This recipe can be made with cucumber, eggplant, tomato, and garlic put into an oven. Provides warm and energized benefits. Hopefully balanced appropriately against other recipes. Make some real ratatouille today. 

## Why's this needed? 
I love eating and making ratatouille in real life, so I decided to try adding it to the game. Also gives some culinary usages to eggplant/cucumber, which have very few recipes. 

## Changelog

```changelog
(u)Roddy
(*)Adds ratatouille to the game
```
unknown (11)

### CODE REVIEW

File: code/modules/food_and_drink/snacks.dm [hunk 1]
```diff
@@ -2892,3 +2892,14 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 	initial_volume = 20
 	initial_reagents = "currypowder"
 	food_effects = list("food_hp_up","food_refreshed","food_warm")
+
+/obj/item/reagent_containers/food/snacks/ratatouille
+	name = "ratatouille"
+	desc = "Stewed and caramalized vegetables. Remy not included."
+	icon = 'icons/obj/foodNdrink/food_meals.dmi'
+	icon_state = "ratatouille"
+	needspoon = 1
```
Comment (TobleroneSwordfish) [inline code/modules/food_and_drink/snacks.dm:2901]:
```suggestion
	needspoon = TRUE
```

**Comment by ZeWaka** [comment]
sprites are very 'eh'

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]
Not sure the sprite is really recognisable as ratatouille, but then I'm not a spriter and have no idea how you'd draw it

**Comment by t-toasted** [comment]
> 

One of the problems I was having with spriting the dish is that it’s a stew that turns a mushy brown when properly cooked, so the alternative I can come up with is doing it in the exact style of the movie ratatouille, if that might fix it

**Comment by WALPVRGIS** [comment]
Hmm, I'm not sold on the latest sprite, it's visually chaotic and a bit hard to decipher. Ratatouille is a tough dish to stylise, theres a lot going on so its gonna be a bit difficult. Definitely possible though, it'll need to exaggerate a lot colour-wise I think rather than going with Brown Stew (or the full dish like the current version you've made). 

Scaling down a photo of the dish suggests a neat direction for the sprite;
image
Something like the first attempt, but pushing those bright colours and slightly more defined Chunks could work hmmm 🤔 Feel free to ping me in spritercord, I'm happy to help figure it out! 😺

**Comment by t-toasted** [comment]
> Hmm, I'm not sold on the latest sprite, it's visually chaotic and a bit hard to decipher. Ratatouille is a tough dish to stylise, theres a lot going on so its gonna be a bit difficult. Definitely possible though, it'll need to exaggerate a lot colour-wise I think rather than going with Brown Stew (or the full dish like the current version you've made).
> 
> Scaling down a photo of the dish suggests a neat direction for the sprite; image Something like the first attempt, but pushing those bright colours and slightly more defined Chunks could work hmmm 🤔 Feel free to ping me in imspriter, I'm happy to help figure it out! 😺

I see the idea! I might ping you on it in a few days. Right now I'll put this on hold for a bit just so I can fiddle with some other projects, but since this just needs one sprite, I can let it simmer a bit.

**Comment by t-toasted** [comment]
El RATOO has been respritted

**Comment by WALPVRGIS** [comment]
!merge_upstream

**Comment by WALPVRGIS** [comment]
Hmm, did a little tweak to give the current sprite outlines, thoughts @t-toasted ?
image
It's still a little weird-looking when not on a plate though, I'm still debating whether this is one of those dishes that will need it's own built-in plate or not 😅

**Comment by t-toasted** [comment]
> Hmm, did a little tweak to give the current sprite outlines, thoughts @t-toasted ?
> image
> It's still a little weird-looking when not on a plate though, I'm still debating whether this is one of those dishes that will need it's own built-in plate or not 😅

Oh wow it looks so much nicer with the outline. Kinda funny, I was literally just thinking “I should look into outlining the rat dish…”
But hMMM yeah if you think it should have a plate then go for it. Also :eyes: pls commit that much nicer outlined version too DHJCKVG

**Comment by ZeWaka** [comment]
@WALPVRGIS closing soon unless update is pushed

**Pull request closed without merging**